contracts/colonyNetwork/ColonyNetworkSkills.sol

```diff
@@ -0,0 +1,460 @@
+/*
```

Comment:
The only thing giving me pause about this is the hectic rebase with #1140 I'm worried it's going to cause. Could we re-order (but stay in the same file), and then do this in a separate PR?

Reply:
I hear that... I don't know if there's a way around a tricky rebase, since even if we keep the re-order in one file the line numbers will still be off. I'm happy to go through and do it, though.

Reply:
Yeah, you're right there in that the big line number changes probably still makes it bad. I guess we just cross that bridge when we come to it.